client/components/EntryFormRow/index.js

```diff
@@ -17,7 +17,7 @@ class EntryFormRow extends Component {
 
   render() {
     return (
-      <tr className="entry-form-row">
+      <tr className="entryFormRow">
```

Comment:
well.. i thought we had css modules set up. It took me by surprise when I saw that css wasn't loading. For css modules we need camel case. I have a chore up for css modules so i left it like this.